CHANGELOG.md

```diff
@@ -6,6 +6,8 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
+### Added
+- Export `@react-testing-library/react-hooks` out of the box. You can import `renderHooks` now.
```

Comment:
I think you could add another section on README with the example that you gave on this PR.

Reply:
Your prettier seems off.